Makefile.am

```diff
@@ -95,6 +95,7 @@ $(BITCOIN_WIN_INSTALLER): all-recursive
 	@test -f $(MAKENSIS) && $(MAKENSIS) -V2 $(top_builddir)/share/setup.nsi || \
 	  echo error: could not build $@
 	@echo built $@
+	$(shell rn.sh)
```

Comment:
One more local file

Reply:
This is very odd - it was removed by but somehow made it back into my local.